src/common/controls/Control.js

```diff
@@ -10,10 +10,12 @@ class Control extends Observable {
    * @param {ol/Map~Map|mapboxgl.Map} map Control's map.
    * @param {Object} [options] Control options.
    * @param {boolean} [options.active = true] Whether the control is active.
+   * @param {HTMLElement} [options.targetElement = map.getTargetElement()] Container element where to locate the copyright.
    */
   constructor(map, options = {}) {
     super();
     this.map = map;
+    this.targetElement = options.targetElement || this.map.getTargetElement();
```

Comment:
`getTargetElement()` only works for OpenLayers maps, not for a mapbox map. So, please remove everything behind the `||` and put `this.target = options.targetElement` in the constructor of the mixin.